index.bs

```diff
@@ -2878,7 +2883,7 @@ statement=], a [=[RP]=] needs to understand these two aspects of [=attestation=]
 
 1. The <dfn>attestation statement format</dfn> is the manner in which the signature is represented and the various contextual
     bindings are incorporated into the attestation statement by the [=authenticator=]. In other words, this defines the
-    syntax of the statement. Various existing devices and platforms (such as TPMs and the Android OS) have previously defined
+    syntax of the statement. Various existing [=client devices=] (such as TPMs and the Android OS) have previously defined
```

Comment:
hm, I wouldn't use "client devices" here.  suggest "existing components and OS platforms"

Reply:
Great, thanks!